source/chapters/microphones.rst

```diff
@@ -411,3 +411,28 @@ input.
 
 DJ controllers typically do not supply phantom power required for condenser
 microphones. Stand-alone audio interfaces typically do supply phantom power.
+
+Microphone Ducking
+------------------
+Microphone ducking is that effect you hear when you speak through the microphone, and the music volume is reduced by the presence of the voice signal coming through the microphone.
+While using Mixxx with a microphone, you might want the music to automatically fade into the background each time you speak, or you might prefer lowering the volume of the music to a specific level. Microphone ducking helps you achieve this.  The music will 'duck' out of the way of the microphone audio.
+
+.. figure:: ../_static/microphone_ducking.png
```

Comment:
Can you put the screenshot in a `2.2` subdirectory and use the same naming scheme (dashes vs. underscores) like you do in #154?

Reply:
Okay. I will.